cypress/page/menu-content.page.ts

```diff
@@ -1,11 +1,13 @@
 export class MenuContentPage {
-  private tShirtMenuSelector: string;
+  private optionList: string;
 
   constructor () {
-    this.tShirtMenuSelector = '#block_top_menu > ul > li:nth-child(3) > a';
+    this.optionList = '#block_top_menu > ul';
   }
 
   public goToTShirtMenu() {
-    cy.get(this.tShirtMenuSelector).click();
+    cy.get(this.optionList).within(($list) => {
+      cy.get('li').last().click();
```

Comment:
what if the menue change the order of the buttons??? and T-shirt menu came in first??
you can impove this using previosly used methods in this course

Reply:
in fact. make it so i can use this methor for any menu button